ui/app/src/components/BrowseFilesDialog/BrowseFilesDialogUI.tsx

```diff
@@ -63,33 +80,151 @@ export function BrowseFilesDialogUI(props: BrowseFilesDialogUIProps) {
+                  {allowUpload && (
+                    <Tooltip title={<FormattedMessage id="word.upload" defaultMessage="Upload" />}>
+                      <IconButton onClick={onUpload} sx={{ mr: 1 }}>
+                        <UploadFileIcon />
+                      </IconButton>
+                    </Tooltip>
+                  )}
+                  <Divider orientation="vertical" flexItem className={classes.actionsBarDivider} />
+                  <SearchBar
+                    keyword={keyword}
+                    onChange={handleSearchKeyword}
+                    showDecoratorIcon
+                    showActionButton={Boolean(keyword)}
+                    classes={{ root: classes.searchRoot }}
+                  />
+                  <Divider orientation="vertical" flexItem className={classes.actionsBarDivider} />
+                  <Button
+                    id="sort-button"
+                    aria-haspopup="true"
+                    aria-expanded={sortMenuOpen ? 'true' : undefined}
+                    onClick={() => setSortMenuOpen(!sortMenuOpen)}
+                    ref={buttonRef}
+                    sx={{ ml: 1, mr: 2 }}
+                  >
+                    <FilterListIcon sx={{ mr: 1 }} />
+                    <FormattedMessage id="words.sorting" defaultMessage="Sorting" />
+                  </Button>
+                  <Menu
+                    id="basic-menu"
+                    anchorEl={buttonRef.current}
+                    open={sortMenuOpen}
+                    onClose={() => setSortMenuOpen(false)}
+                    MenuListProps={{
+                      'aria-labelledby': 'sort-button'
+                    }}
+                  >
+                    <MenuItem>
+                      <FormControl fullWidth>
+                        <InputLabel>
+                          <FormattedMessage id="BrowseFilesDialog.sortBy" defaultMessage="Sort By" />
+                        </InputLabel>
+                        <Select
+                          fullWidth
+                          value={searchParameters.sortBy}
+                          onChange={({ target }) => {
+                            setSearchParameters({
+                              sortBy: target.value
+                            });
+                          }}
+                          size="small"
+                          className={classes.sortingSelect}
+                          label={<FormattedMessage id="BrowseFilesDialog.sortBy" defaultMessage="Sort By" />}
+                        >
+                          <MenuItem value={'_score'}>
+                            <FormattedMessage id="words.relevance" defaultMessage="Relevance" />
+                          </MenuItem>
+                          <MenuItem value={'internalName'}>
+                            <FormattedMessage id="words.name" defaultMessage="Name" />
+                          </MenuItem>
+                          {sortKeys.map((name, i) => (
```

Comment:
Formatting looks broken in github. Check you've run prettier and there are no issues in the app build

Reply:
Does it look correct now? I ran prettier manually but I saw no difference